capstone/src/test/javascript/graphsTest.js

```diff
@@ -0,0 +1,15 @@
+describe('Data Table Construction', function() {
```

Comment:
Same comment about using arrow function. Same comment for other instances below.